std/zip.d

```diff
@@ -38,6 +39,26 @@ void main(string[] args)
         assert(am.expandedData.length == am.expandedSize);
     }
 }
+
+// Create and write new zip file.
+import std.file: write;
+import std.string: representation;
+
+void main(string[] args)
+{
+    char[] data = cast(char[])"Test data.\n";
```

Comment:
This is unsafe and probably shouldn't be in an example.  Maybe use `"...".dup` to create a mutable string?